vcf2fhir/fhir_helper.py

```diff
@@ -300,6 +303,158 @@ def add_variant_obv(self, record, ref_seq, ratio_ad_dp):
         )
         observation_dv.component = []
 
+        if record.is_sv:
+            dna_chg = get_dna_chg(record.INFO['SVTYPE'])
+            observation_dv_component1 = observation.ObservationComponent()
+            observation_dv_component1.code = concept.CodeableConcept(
+                {
+                    "coding": [
+                        {
+                            "system": "http://loinc.org",
+                            "code": "48019-4",
+                            "display": "DNA Change Type"
+                        }
+                    ]
+                }
+            )
+            observation_dv_component1\
+                .valueCodeableConcept = concept.CodeableConcept(
+                    {
+                        "coding": [
+                            {
+                                "system": ("http://sequenc" +
+                                           "eontology.org"),
+                                "code": dna_chg['CODE'],
+                                "display": dna_chg['DISPLAY']
+                            }
+                        ]
+                    }

```

Comment:
@srgothi92, I'm not sure if this would ensure the order of the variant components.

What I did to reduce the complexity of the code was: 

- First I changed the **variable names** for all the **variant components** from `observation_dv_component{component_number}` to a name that's related to it like [`dna_change_type_component`](https://github.com/Rohan-cod/vcf2fhir/blob/structural-variants/vcf2fhir/fhir_helper.py#L195).

- Second I created a list of the variant components(in order) [**here**](https://github.com/Rohan-cod/vcf2fhir/blob/structural-variants/vcf2fhir/common.py#L14).

- Third, I used that list to add components that are declared(checked using the `locals()` dictionary) to the `observation_dv.component` [list](https://github.com/Rohan-cod/vcf2fhir/blob/structural-variants/vcf2fhir/fhir_helper.py#L191).

This would also make it easier to add a **new variant component** in the future. We just have to update the **variant components list** [**here**](https://github.com/Rohan-cod/vcf2fhir/blob/structural-variants/vcf2fhir/common.py#L14) by adding the name of the component at the correct position and add the code to create the component in the `add_variant_obv` [function](https://github.com/Rohan-cod/vcf2fhir/blob/structural-variants/vcf2fhir/fhir_helper.py#L156).

This made the logic and maintainability, a lot simpler.

I have also added a comment[ just above](https://github.com/Rohan-cod/vcf2fhir/blob/structural-variants/vcf2fhir/fhir_helper.py#L359) the code to append variant components.